README.md

```diff
@@ -1,2 +1,62 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+PROGRAM Terminal
+
+Init Terminal
+
+Guesses = 4
+
+Correct Word = Marks
+
+Words = Starks, Stake, Helps, Empty, Marks
+
+Print “You have 4 guesses”
+
+Guess = Input Guess
+
+
+If chosen word has no characters right 
```

Comment:
The worksheet asked you to provide pseudocode for an algorithm to calculate the number of correct letters. Be sure to read the specification carefully in future to ensure you are doing the task that's required!